Theory of Computing.js

```diff
@@ -9,10 +9,10 @@
 	"inRepository": true,
 	"translatorType": 4,
 	"browserSupport": "gcv",
-	"lastUpdated": "2017-07-03 10:55:03"
+	"lastUpdated": "2019-06-10 22:52:50"
 }
 
-/*****
+/** ***
```

Comment:
Let us use here simply
```js
/*
...
*/
```
and delete the other asterisks in the beginning and the end.